content/post/roadmap.md

```diff
@@ -0,0 +1,65 @@
+
+You may end up tying your work back to, say, 2 of your organization's 3 strategic goals for the year. That's not in and of itself bad, but should be called out. This exercise reduces blind spots and forces you to think about how you can support all aspects of the mission.
+
+### Set up the scaffolding
+
+How do you start? After outlining our Data team mission and values (distinct from the broader company) I like to make explicit what areas of focus we're responsible for. For us, that's Data infrastructure, BI, analytics and strategic projects, Data Science and Clinical Data products. This helps to compartmentalize projects, especially those that are cross functional.
+
+You may find it more natural to split your road map by the various teams you support, e.g. Marketing, Product, Sales, etc... I prefer the former because it captures both technical differences that are relevant to the team (e.g. data analysts will skew heavily towards BI, data engineers towards data infrastructure, etc...) and "Analytics and Strategic projects" encapsulate projects that are cross functional (e.g. Marketing, Sales, Finance).
+
+### Sync with functional leaders 
+
+After putting down some projects that fit into the relevant areas of focus, set up time with key functional leaders and get their input. Hopefully you sequence this exercise at a time in which they are also thinking about their goals for the year. Have them help you fill in blind spots and loosely prioritize their projects. You may know of four Operations projects that are critical for 2019, but the COO may have a different idea on what the sequencing should be. Don't get bogged down by timing at this point, but getting a rank-ordered list of projects - at least within each functional stakeholder - is very helpful. For example, it may be too early to understand if Finance's pricing model should be prioritized over the Customer Support team's call queue algorithm, but its helpful to identify if automating Finance's monthly billing report is more important than their pricing model.
+
+Functional leaders will also be helpful in identifying cross team dependencies. You may have grand plans for implementing [Airflow](https://airflow.apache.org/), but if you lack Data Engineering resources, you may have to rethink your plan. Similarly you may have an idea for a great data-driven product feature, but if Product isn't prioritizing that in their roadmap, it may drop to the bottom of your priority, even though you think it's a valuable feature.
+
+### Put stakes in the ground 
+
```

Comment:
Do you recommend attempting to do some sort of ROI calculation or avoiding ROI all together?

Reply:
Two thoughts:
1. A rank-ordered list will hopefully have some approximation of ROI embedded within in. 
2. Any level of ROI detail greater than the rank-ordered list is probably unnecessary and ultimately inaccurate at this stage. For sake of expediency and rallying around projects/objectives, probably best to save ROI calcs for when you're closer to resource allocation time.